layers/+completion/ivy/README.org

```diff
@@ -77,17 +77,18 @@ In case you don't want to read everything, at least familiarise with
 
 Some useful key bindings are presented in the following table.
 
-| Key binding | Description                                                                                          |
-|-------------+------------------------------------------------------------------------------------------------------|
-| ~RET~       | call default action on current candidate                                                             |
-| ~M-RET~     | the same as ~RET~ but doesn't close completion minibuffer                                            |
-| ~C-M-j~     | use current input immediately (this can be used to create a new file in Find File)                   |
-| ~TAB~       | complete partially                                                                                   |
-| ~M-o~       | show the list of valid actions on current candidate (then press any of described keys to execute it) |
-| ~C-M-o~     | the same as ~M-o~ but doesn't close completion minibuffer                                            |
-| ~C-'~       | use avy to quickly select completion on current page (sometimes faster than using arrows)            |
-| ~<ESC>~     | close minibuffer                                                                                     |
-| ~C-M-k~     | kill buffer (in =ivy-switch-buffer= (~SPC b b~))                                                     |
+| Key binding | Description                                                                                                                                         |
+|-------------+-----------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
This looks like the formatting went completely crazy, please double check

Reply:
Okay, so this generally happens when I use `org-cycle` (i.e. `TAB` key) to automatically format the table. So I should always correct the formatting manually in text-mode? Or do you have some advice for how to do it automatically?

Reply:
Nope, I write the documentation in vanilla vim, as my spacemacs heavily reformats buffers before saving, however as the documentation is also used as basis for generating html and markdown we have to follow certain formatting rules.

Reply:
Ah thanks! Will use vanilla Emacs for that then :+1:

Reply:
Pressing `TAB` in a key binding table has worked for me in Spacemacs,
to automatically fit the table to the longest description line.

Reply:
Seems like with a new/clean dotfile it works correctly indeed.